group_vars/mysql-slave/public.yml

```diff
@@ -0,0 +1,31 @@
+MYSQL_SERVER_DOMAIN: "{{ inventory_hostname }}"
+
+MYSQL_SERVER_BACKUP_DIR: '/var/lib/mysql/backup'
+TARSNAP_BACKUP_PRE_SCRIPT: "/usr/local/sbin/backup-pre.sh"
+TARSNAP_BACKUP_POST_SCRIPT: "/usr/local/sbin/backup-post.sh"
+TARSNAP_BACKUP_FOLDERS: "{{ MYSQL_SERVER_BACKUP_DIR }}"
+TARSNAP_ARCHIVE_NAME: "{{ MYSQL_SERVER_DOMAIN }}"
+
+mysql_root_password_update: true
+
+mysql_packages:
+  - mysql-server-5.6
+  - mysql-client-5.6
+
+mysql_server_id: "{{ MYSQL_SERVER_ID | default('99') }}"
```

Comment:
This ID needs to be unique across the replica set, right?  In that case, I suggest removing the default to avoid unintended conflicts.

Reply:
Yes it does. I'll remove the default.